docs/connector/08-01-establish-secure-connection-with-compass.md

```diff
@@ -0,0 +1,88 @@
+---
```

Comment:
The convention is to use H1 for titles of docs kept on Github, and matadata for docs that are displayed on website. Let's discuss this.